### PR TITLE
[WORKFLOW] Trigger build_conf on various change across GUI

### PR DESCRIPTION
**Pull request by ELDiablO59152**
### Fixed
- [WORKFLOW] Trigger build_conf on various changes

### CODE REVIEW

File: vulture_os/system/config/views.py [hunk 1]
```diff
@@ -107,10 +106,12 @@ def render_form(**kwargs):
         if "internal_tenants" in form.changed_data:
             Cluster.api_request("services.rsyslogd.rsyslog.configure_pstats")
             Cluster.api_request("services.rsyslogd.rsyslog.restart_service")
-        if "portal_cookie_name" in form.changed_data:
+        if "portal_cookie_name" or "public_token" in form.changed_data:
             # Reload Frontends with authenticated Workflows: session checks must be updated with the new cookie's name
             for frontend in Frontend.objects.filter(workflow__authentication__isnull=False).distinct():
-                frontend.reload_conf()
+                for node in frontend.reload_conf():
+                    node.api_request("workflow.workflow.build_conf", workflow.pk)
```
Comment (frikilax) [inline vulture_os/system/config/views.py:113]: `workflow` is undefined!

File: CHANGELOG [hunk 2]
```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [Unreleased]
+### Fixed
+- [WORKFLOW] Trigger build_conf on various change
```
Comment (frikilax) [inline CHANGELOG:10]:
```suggestion
- [WORKFLOW] Trigger build_conf on various changes
```

File: vulture_os/system/config/views.py (hunk 1 above)
Comment (ELDiablO59152) [reply vulture_os/system/config/views.py:121]: You are right, I fixed it and added vulture's idps reload to it

File: CHANGELOG (hunk 2 above)
Comment (ELDiablO59152) [reply CHANGELOG]: Changed

File: vulture_os/system/config/views.py [hunk 3]
```diff
@@ -107,10 +114,14 @@ def render_form(**kwargs):
         if "internal_tenants" in form.changed_data:
             Cluster.api_request("services.rsyslogd.rsyslog.configure_pstats")
             Cluster.api_request("services.rsyslogd.rsyslog.restart_service")
-        if "portal_cookie_name" in form.changed_data:
-            # Reload Frontends with authenticated Workflows: session checks must be updated with the new cookie's name
-            for frontend in Frontend.objects.filter(workflow__authentication__isnull=False).distinct():
-                frontend.reload_conf()
+        if "portal_cookie_name" or "public_token" in form.changed_data:
```
Comment (frikilax) [inline vulture_os/system/config/views.py:117]:
```suggestion
        if "portal_cookie_name" or "public_token" in form.changed_data:
```
Nope, doesn't work like that!

Comment (ELDiablO59152) [reply vulture_os/system/config/views.py:117]: You are absolutely right on that ! Thank you

**Review by frikilax — DISMISSED** [review]

**Review by frikilax — APPROVED** [review]

**Pull request merged**